src/test/java/org/jenkins/plugins/statistics/gatherer/listeners/ItemStatsListenerTest.java

```diff
@@ -0,0 +1,89 @@
+package org.jenkins.plugins.statistics.gatherer.listeners;
+
+import hudson.model.Build;
+import hudson.model.FreeStyleProject;
+import hudson.model.Item;
+import org.jenkins.plugins.statistics.gatherer.model.build.BuildStats;
+import org.jenkins.plugins.statistics.gatherer.model.job.JobStats;
+import org.jenkins.plugins.statistics.gatherer.util.LogbackUtil;
+import org.jenkins.plugins.statistics.gatherer.util.PropertyLoader;
+import org.jenkins.plugins.statistics.gatherer.util.RestClientUtil;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mockito.ArgumentCaptor;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({PropertyLoader.class, RestClientUtil.class, LogbackUtil.class})
+@PowerMockIgnore({"javax.crypto.*"})
+public class ItemStatsListenerTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
maybe use a more meaningful name for clarity?

Reply:
Sure.